perception/radar_fusion_to_detected_object/package.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>radar_fusion_to_detected_object</name>
+  <version>0.0.0</version>
+  <description>radar_fusion_to_detected_object</description>
+  <maintainer email="scepter914@gmail.com">scepter914</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
+  <build_depend>autoware_cmake</build_depend>
+
+  <depend>autoware_auto_perception_msgs</depend>
+  <depend>geometry_msgs</depend>
+  <depend>rclcpp</depend>
+  <depend>rclcpp_components</depend>
+  <depend>std_msgs</depend>
+  <depend>tf2</depend>
```

Comment:
Do you need this dependency?
I think you're not using it.

Reply:
fix

Reply:
Can you please write the commit where this was fixed?

Reply:
Here
https://github.com/scepter914/autoware.universe/commit/c8a936c536b64e7e80e8821db6cd9bd00c785a6d

Reply:
Fixed [here](https://github.com/scepter914/autoware.universe/commit/c8a936c536b64e7e80e8821db6cd9bd00c785a6d)